### PR TITLE
VZ-10099: format the output timestamp to add timezone offset

### PR DESCRIPTION
**Pull request by karan56625**
format the output timestamp to add timezone offset

### CODE REVIEW

**Review by adiforluls — COMMENTED** [review]
Could you paste the timestamp format without this change, the timestamp format with this change and the timestamp format we get in fluentd

File: platform-operator/thirdparty/charts/fluent-operator/templates/verrazzano/fluentbit-output-filter.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{- if .Values.fluentbit.enable -}}
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFilter
+metadata:
+  name: vz-output-filter
```
Comment (adiforluls) [inline platform-operator/thirdparty/charts/fluent-operator/templates/verrazzano/fluentbit-output-filter.yaml:5]: Maybe the name should be something like `vz-timestamp-modifier` or `vz-timestamp-format-filter` to convey what this filter is for just by name

**Comment by karan56625** [comment]
> Could you paste the timestamp format without this change, the timestamp format with this change and the timestamp format we get in fluentd

fluentd:
"@timestamp": "2023-06-15T06:47:03.965000000+00:00"
fluentbit:
"@timestamp": "2023-06-15T09:37:37.000Z"
fluentbit(after this PR)
"@timestamp": "2023-06-20T16:00:20.973099067+00:00"

File: platform-operator/thirdparty/charts/fluent-operator/templates/verrazzano/fluentbit-output-filter.yaml (hunk 1 above)
Comment (karan56625) [reply platform-operator/thirdparty/charts/fluent-operator/templates/verrazzano/fluentbit-output-filter.yaml]: renamed it to vz-timestamp-format-filter. that should be fine?

**Review by adiforluls — APPROVED** [review]

**Pull request merged**